Server/shop/src/main/java/com/cashmanager/server/shop/Order.java

```diff
@@ -0,0 +1,48 @@
+package com.cashmanager.server.shop;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.proxy.HibernateProxy;
+
+import java.util.Date;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@Builder
+@AllArgsConstructor
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column( nullable = false)
+    // GenerationType instructs that a UUID for the entity should be generated automatically for us by the persistence provider.
+    private UUID order_id;
```

Comment:
We already know that we are in an Order, this is not a foreign key

```suggestion
    private UUID id;
```